src/components/Login.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-container fluid fill-height class="pa-0 ma-0">
+    <v-layout align-center justify-center>
+      <v-flex text-xs-center fill-height>
```

Comment:
fill-height is not a valid attribute according to the docs

Reply:
neither was text-xs-center, both fixed